### PR TITLE
Minor changes to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by baroquebobcat**
Some minor copy changes around running CI tests locally.
* Add descriptions for scripts.
* Update TARGET_SET to point at a declared target.
* Break running individual tests out into it's own short section.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -130,15 +130,26 @@ test sources, resources, test resources, generated sources, etc).
...
+* `./scripts/run-tests-ci.sh` Runs tests using the setup from the previous commands.
 
-* Individual test or target set can be run as the following, and the parameters are also used by .travis.yml:
-  * `TEST_SET=integration ./scripts/run-tests-ci.sh --test-junit-test=com.twitter.intellij.pants.integration.OSSPantsJavaExamplesIntegrationTest#testJaxb`
+### Running individual tests
+
+Individual test or target set can be run as the following, and the parameters are also used by .travis.yml:
+
+```
+TEST_SET=jvm-integration \./scripts/run-tests-ci.sh --test-junit-test=com.twitter.intellij.pants.integration.OSSPantsJavaExamplesIntegrationTest#testJaxb
```
Comment (wisechengyi) [inline CONTRIBUTING.md]: is `\./scripts/run-tests-ci.sh` intentional?

Comment (baroquebobcat) [inline CONTRIBUTING.md]: Sort of. Missed a new line. I meant to make it:
```
TEST_SET=jvm-integration  \
./scripts/run-tests-ci.sh \
--test-junit-test=com.twitter.intellij.pants.integration.OSSPantsJavaExamplesIntegrationTest#testJaxb
```

Comment (wisechengyi) [inline CONTRIBUTING.md]: sure. feel free to add it then merge

Comment (wisechengyi) [inline CONTRIBUTING.md]: ok got it. github has the feature edit the PR directly on web, which is pretty cool.

**Comment by baroquebobcat** [comment]
That's cool.

**Pull request merged**